Signal/src/network/PushManager.m

```diff
@@ -147,16 +148,16 @@ - (void)application:(UIApplication *)application
 
         if (threadId) {
             TSThread *thread = [TSThread fetchObjectWithUniqueID:threadId];
-            TSOutgoingMessage *message =
-                [[TSOutgoingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
-                                                    inThread:thread
-                                                 messageBody:responseInfo[UIUserNotificationActionResponseTypedTextKey]];
-            [self.messageSender sendMessage:message
+            NSString *replyText = responseInfo[UIUserNotificationActionResponseTypedTextKey];
+
+            [ThreadUtil sendMessageWithText:replyText
+                inThread:thread
+                messageSender:self.messageSender
```

Comment:
There's a lot of places we call `sendMessage:`, e.g. in many places we send "group update" and "leave group" messages.  Should these be moved to using some `ThreadUtil` method that preserves this policy or does that not matter in these cases?